Gopkg.toml

```diff
@@ -90,8 +90,9 @@
   name = "github.com/juju/naturalsort"
 
 [[constraint]]
-  revision = "88a4c6ac59c1950f118f058d920204c9182b9927"
+  branch = "workload-crtl-series"
   name = "github.com/juju/os"
+  source = "github.com/SimonRichardson/os"
```

Comment:
This is only used during development and will alter once everything has landed and settled.